packages/dialect-react-ui/components/Chat/screens/CreateThreadPage/ActionCaption.tsx

```diff
@@ -0,0 +1,63 @@
+import { useDialectSdk } from '@dialectlabs/react-sdk';
+import type { DialectSdkError } from '@dialectlabs/sdk';
+import clsx from 'clsx';
+import { A, P } from '../../../common/preflighted';
+import { useTheme } from '../../../common/providers/DialectThemeProvider';
+
+export default function ActionCaption({
+  creationError,
+  encrypted,
+}: {
+  encrypted: boolean;
+  creationError: DialectSdkError | null;
+}) {
+  const { textStyles, xPaddedText } = useTheme();
+  const {
+    info: { apiAvailability },
+  } = useDialectSdk();
+
+  if (creationError && creationError.type !== 'DISCONNECTED_FROM_CHAIN') {
```

Comment:
Let's make an enum for these types.

Reply:
we haven't worked on react-ui refactoring yet. all Errors are as classes now, but it should be replaced in ui, which was out of scope for now